### PR TITLE
Add models for deviceclass and deviceconfig

### PR DESCRIPTION
**Pull request by marcelveldt**
add some missing models

### CODE REVIEW

**Review by balloob — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]

File: zwave_js_server/model/node.py [hunk 1]
```diff
@@ -42,9 +45,9 @@ def ready(self) -> bool:
         return self.data.get("ready")
 
     @property
-    def device_class(self) -> dict:  # XXXXXX NEEDS CLASS
+    def device_class(self) -> dict:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/node.py:48]:
```suggestion
    def device_class(self) -> DeviceClass:
```

File: zwave_js_server/model/node.py [hunk 2]
```diff
@@ -127,9 +130,9 @@ def location(self) -> str:
         return self.data.get("location")
 
     @property
-    def device_config(self) -> dict:  # XXXXXX NEEDS CLASS
+    def device_config(self) -> dict:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/node.py:133]:
```suggestion
    def device_config(self) -> DeviceConfig:
```

**Comment by marcelveldt** [comment]
good catch about the typing. missed that

**Comment by MartinHjelmare** [comment]
Tests fail. I'm looking at it.

**Comment by MartinHjelmare** [comment]
@marcelveldt Please check and go ahead and merge.

**Pull request merged**